hi-ml/src/health_ml/utils/split_dataset.py

```diff
@@ -275,7 +276,7 @@ def from_groups(df: pd.DataFrame,
 
     @staticmethod
     def get_df_from_ids(df: pd.DataFrame, ids: Sequence[str],
-                        subject_column: str = "") -> pd.DataFrame:
+                        subject_column: Optional[str] = "") -> pd.DataFrame:
```

Comment:
For the rest of the codebase, we use "" to indicate missing. None and "" both evaluate to False, so you can write 'if not mystring:...'